README.md

```diff
@@ -159,7 +176,7 @@ Default input namespace [SCRC]:
 Initialised empty fair repository in /home/joebloggs/Documents/AnalysisProject/.fair
 ```
 
-**Generated `config.yaml`**
+#### Generated `config.yaml`
```

Comment:
I used `**` because I think GH does not recognise 4th order headers (these won't be bold if I recall)

Reply:
Maybe they've updated the GH renderer, but it seems to work now - https://github.com/FAIRDataPipeline/FAIR-CLI/tree/rr/fixes ?